sklearn/linear_model/tests/test_passive_aggressive.py

```diff
@@ -69,24 +71,38 @@ def project(self, X):
 def test_classifier_accuracy():
     for data in (X, X_csr):
         for fit_intercept in (True, False):
-            clf = PassiveAggressiveClassifier(C=1.0, n_iter=30,
-                                              fit_intercept=fit_intercept,
-                                              random_state=0)
-            clf.fit(data, y)
-            score = clf.score(data, y)
-            assert_greater(score, 0.79)
+            for average in (False,True):
+                clf = PassiveAggressiveClassifier(C=1.0, n_iter=30,
+                                                  fit_intercept=fit_intercept,
+                                                  random_state=0,
+                                                  average=average)
+                clf.fit(data, y)
+                score = clf.score(data, y)
+                assert_greater(score, 0.79)
+                if average:
+                    assert_true(hasattr(clf, 'average_coef_'))
```

Comment:
I don't know what kind of relevant characteristics these weights have that could be checked.
The check on the score in line 81 is an indirect check on them that they produce sound classifications.